style.css

```diff
@@ -1,3 +1,58 @@
-body {
+/* body {
   background-color: white;
+} */
+
+#bookss {
+  /* border: 2px red solid; */
+  display: flex;
+  width: 70%;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  margin: 0 auto;
+}
+
+#dynamic {
+  border: black 2px solid;
+}
+
+hr {
+  width: 40%;
+  height: 2px;
+  background-color: black;
+}
+
+.table {
+  /* display: flex; */
+
+  /* border: 2px red solid; */
```

Comment:
- [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.